jenkins/jobs/cd/tiflow_docker.groovy

```diff
@@ -0,0 +1,31 @@
+pipelineJob('tiflow-docker') {
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath('jenkins/pipelines/cd/tiflow-docker.groovy')
+            scm {
+                git{
+                    remote {
+                        url('git@github.com:PingCAP-QE/ci.git')
```

Comment:
It is recommended to modify it to download via the https URL, as this repo is public.

Reply:
https://github.com/PingCAP-QE/ci/blob/main/jobs/pingcap/tidb/latest/ghpr_build.groovy#L23-L35